src/Transports/MassTransit.EventHubIntegration/EventHubIntegration/Checkpoints/PendingConfirmationCollection.cs

```diff
@@ -16,9 +17,9 @@ public PendingConfirmationCollection(string eventHubName)
             _confirmations = new ConcurrentDictionary<long, IPendingConfirmation>();
         }
 
-        public IPendingConfirmation Add(ProcessEventArgs eventArgs)
+        public IPendingConfirmation Add(ProcessEventArgs eventArgs, CancellationToken cancellationToken)
```

Comment:
This feels like something the collection should own, vs. each individual confirmation.